docs/DeveloperGuide.md

```diff
@@ -6,33 +6,227 @@
+| v1.0    | user     | be able to label my destinations                               | identify the type of activity easily                                                      |
+| v1.0    | user     | be able to add and remove activities from my planner           | easily make changes to my current activity list                                           | 
+| v1.0    | user     | be able to add tags or categories to my activities             | organise them based on my preferences                                                     |
+| v1.0    | user     | be able to check and uncheck my travel activities              | easily mark the activities that I have done if I do not follow the sequence of activities |
+| v1.0    | user     | have a help option                                             | easily identify all the commands in the chatbot                                           |
+| v1.0    | user     | be able to search for an activity already recorded in the list | easily retrieve exisiting plans and not have double inputs                                |
+| v2.0    | user     | maintain my expense records                                    | make travel plans according to my budget                                                  |
+| v2.0    | user     | be able to see the travel durations                            | plan my activities more efficiently                                                       |
+| v2.0    | user     | be able to search for an activity based on its type or tag     | find specific places in my list more efficiently                                          |
+| v2.0    | user     | be able to delete a destination from my records                | remove redundant records that I do not need                                               |
+| v2.0    | user     | be able to edit and update existing plans                      | efficiently update exisiting plans without having to delete and add plans                 |
+| v2.0    | user     | have some form of QOL considerations like warning messages     | know what are the errors that I have made in the inputs                                   |
+
 
 ## Non-Functional Requirements
 
 {Give non-functional requirements}
 
 ## Glossary
 
-* *glossary item* - Definition
+* Mainstream OS: Windows, Linux, MacOS
 
 ## Instructions for manual testing
 
-{Give instructions on how to do a manual product testing e.g., how to load sample data to be used for testing}
+
+
+- To add a general travel activity into the list
+
+Example: `add visit Paris /date 2024-12-12 /duration 2 weeks`
+
+Expected: A new general activity is added
+
+- To add an accommodation activity into the list
+
+Example: `accommodation hotel /date 2024-12-12 /duration one night`
+
+Expected: A new accommodation activity type is added
+
+- To add a food activity into the list
+
+Example: `food prata /date 2024-12-12 /duration 1 hour`
```

Comment:
Could you explore ways to adhere more closely to coding standards and enhance code quality, such as adding : before classes?